src/main/java/core/basesyntax/model/figure/Circle.java

```diff
@@ -1,6 +1,15 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class Circle extends Figure {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
Since we mark `Figure` with annotation `MappedSuperClass` it is not necessary to mark any sub classes with any annotation as they will inherit all fields respectively.